test/test_integration.py

```diff
@@ -62,6 +62,28 @@ def test_ontology_without_schema_config(core_no_schema):
     assert isinstance(core_no_schema._ontology._nx_graph, nx.DiGraph)
 
 
+def test_show_full_ontology_without_schema_config(core_no_schema):
```

Comment:
@nilskre I see two issues with doing it like this:
- this test repeats a lot of code of the above test, including two identical calls to an online resource (which should be kept to an absolute minimum)
- testing the ontology structure does not need to be an integration test

To fix the first, one could create a fixture that is used by both tests, but I would not do that and rather go for implementing a unit test based on one of the on-disk ontology files that are used in the ontology test module. All you need is an `Ontology` class instance without a schema definition, you can call the `show_ontology_structure` method of that class directly.

Reply:
Yes, true. Thanks for the feedback. I will update it (probably tomorrow).

Reply:
The test is now adapted accordingly. 
When reviewing don't be confused, that there is still a BioCypher object initialized (with a local ontology). This is necessary, because the line, which needs to be tested is in [_core.py](https://github.com/biocypher/biocypher/blob/d3195ea33d2e7b2bd516336dc1bbf38999a8221e/biocypher/_core.py#L184). I have tested it locally with the old code (new test fails) and the new code (new test passes).